### PR TITLE
added sandbox property to iframe

### PR DESCRIPTION
**Pull request by elias-summermatter**

### CODE REVIEW

**Review by michael-scheurer — DISMISSED** [review]

File: packages/editor/src/client/utility.ts [hunk 1]
```diff
@@ -207,6 +207,14 @@ export function validateURL(url: string) {
   return false
 }
 
+export function flattenArray(array: unknown[] | DOMTokenList) {
```
Comment (tomaszdurka) [inline packages/editor/src/client/utility.ts:210]: Not sure why we need this? This works exactly like `array.join(" ")`, let's use that in tsx.

**Review by tomaszdurka — CHANGES_REQUESTED** [review]

File: packages/editor/src/client/utility.ts (hunk 1 above)
Comment (elias-summermatter) [reply packages/editor/src/client/utility.ts:210]: Since its a DOMTokenList there is no join("") function.....

**Comment by elias-summermatter** [comment]
> If you want hotfix on 2.15, we can't merge to master.

Sorry what we want is to hotfix 3.0.0-alpha.1 or we just make a normal release since we have not done one since 3 weeks

File: packages/editor/src/client/utility.ts (hunk 1 above)
Comment (tomaszdurka) [reply packages/editor/src/client/utility.ts:210]: OK!

**Review by tomaszdurka — CHANGES_REQUESTED** [review]

**Comment by tomaszdurka** [comment]
If that's the case we can merge to master sure.

File: packages/editor/package.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
     "watch:client": "webpack-dev-server --mode development --hot --inline",
     "watch:server": "run-p watch:server:*",
     "watch:server:start": "nodemon -r dotenv/config --inspect=0.0.0.0:9230 ./dist/server/index.js",
-    "watch:server:build": "tsc -b ./src/shared ./src/server --watch",
```
[the remark this answers is not in the document]
Comment (elias-summermatter) [reply packages/editor/package.json:40]: Was just fo developing

**Review by penk85 — APPROVED** [review]

**Review by camilleory — APPROVED** [review]

**Review by tomaszdurka — APPROVED** [review]

**Pull request merged**